pkgs/tools/inputmethods/fcitx5/fcitx5-mozc.nix

```diff
@@ -1,129 +1,145 @@
-{ lib, clangStdenv, fetchFromGitHub, fetchurl, fetchpatch
-, python3Packages, ninja, pkg-config, protobuf, zinnia, qt5, fcitx5
-, jsoncpp, gtest, which, gtk2, unzip, abseil-cpp, breakpad, nixosTests }:
-let
-  inherit (python3Packages) python gyp six;
-  utdic = fetchurl {
-    url = "https://osdn.net/downloads/users/39/39056/mozcdic-ut-20220904.tar.bz2";
-    sha256 = "sha256-pmLBCcw2Zsirzl1PjYkviRIZoyfUz5rpESeABDxuhtU=";
-  };
-  japanese_usage_dictionary = fetchFromGitHub {
-    owner = "hiroyuki-komatsu";
-    repo = "japanese-usage-dictionary";
-    rev = "e5b3425575734c323e1d947009dd74709437b684";
-    sha256 = "0pyrpz9c8nxccwpgyr36w314mi8h132cis8ijvlqmmhqxwsi30hm";
-  };
-  zipcode_rel = "202011";
-  jigyosyo = fetchurl {
-    url = "https://osdn.net/projects/ponsfoot-aur/storage/mozc/jigyosyo-${zipcode_rel}.zip";
-    sha256 = "j7MkNtd4+QTi91EreVig4/OV0o5y1+KIjEJBEmLK/mY=";
-  };
-  x-ken-all = fetchurl {
-    url =
-      "https://osdn.net/projects/ponsfoot-aur/storage/mozc/x-ken-all-${zipcode_rel}.zip";
-    sha256 = "ExS0Cg3rs0I9IOVbZHLt8UEfk8/LmY9oAHPVVlYuTPw=";
-  };
-
-in clangStdenv.mkDerivation {
+{
+  bazel_7,
+  buildBazelPackage,
+  fcitx5,
+  fetchFromGitHub,
+  gettext,
+  lib,
+  mozc,
+  nixosTests,
+  pkg-config,
+  python3,
+  unzip,
+}:
+
+buildBazelPackage {
   pname = "fcitx5-mozc";
-  version = "2.26.4220.102";
+  version = "2.30.5544.102";
```

Comment:
Just out of curiosity, does the version need to be in sync with mozc?
If so, maybe when there's an update to mozc we should wait until `fcitx-mozc` can be updated too, and update all at once.

Reply:
It's already in sync with the current `mozc` package in nixpkgs though:

https://github.com/NixOS/nixpkgs/blob/d74f0ecb14965cde6de0a0718509337ddac9058d/pkgs/by-name/mo/mozc/package.nix#L21

Though I believe that it should be generally ok even if it is a bit out-of-sync.